test/requirements.txt

```
@@ -0,0 +1,2 @@
+selenium
+pytest
```

本質ではないですが、Pythonは勝手に壊れることがあるので、pipenvもしくはpeotryを利用して、ライブラリのバージョンを固定出来るならしておきたいです。
使い方が分からない場合は、一旦このまま、バージョンの指定だけしておくことにして、マージできた後に、別PRで対応しようかなと思っています！
```suggestion
selenium == 4.3.*
pytest == 7.1.*
```

---

これは確かにそうでして、Pythonが勝手に壊れることは今回は特に考えていませんでした。(私も昔勝手に壊れました)
モジュールのインストールにpoetryを使用している例をよく見るので、もしよろしければ使用するコードを書いていただけましたら幸いです。
今回はバージョン指定にとどめておきます。